data/data-pipeline/data_pipeline/tests/sources/example/data/input.csv

```diff
@@ -1,16 +1,16 @@
-GEOID10_TRACT,Input Field 1
-06007040300,2
-06001020100,6.1
-06007040500,-7.8
-15001021010,12
-15001021101,12.05524783
-15007040603,13.51417578
-15007040700,13.11988976
-15009030100,13.60946983
-15009030201,13.73235164
-15001021402,14.73305116
-15001021800,16.60833857
-15009030402,16.002535
-15009030800,15.34818251
-15003010201,14.58788769
-15007040604,14.27704917
\ No newline at end of file
+,GEOID10_TRACT,Input Field 1
```

Comment:
Hmm - are we sure we want (1) the index and (2) to drop leading zeroes (i.e., "06007040300" has now gone to "6007040300" here) here?

Reply:
these are both super good points, yikes